packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -129,30 +133,43 @@ export const makeVaultManager = (
   );
 
   /**
-   *
-   * @param {[key: string, vaultKit: InnerVault]} record
+   * @param {Iterable<[key: string, vaultKit: InnerVault]>} recordEntries
```

Comment:
`recordEntries` is confusing to me. I alternate between reading `record` as a noun and a verb. Can they be `vaultEntries` or something like that?